### DISCUSSION TITLE
Created tables are not persisting when I logout and login

### DISCUSSION THREAD

**Discussion by hariprasadms** [AntlerVC/firetable#460] [Support / Q&A] [answered]
Hi, I have deployed the firetable to firebase hosting and linked to a project. Deployment works fine. but major issues is all my created tables are not persisting in the front end. every-time when I logout and login, I do not see the created tables. it always shows blank. 

Please help...

**Comment by harinij** [comment, marked as answer]
Hi @hariprasadms, this issue is resolved in the rc branch for the tables to be visible to ADMIN users by default in the main dashboard. We plan on releasing the rc branch to the master later this week

**Reply by hariprasadms** [reply]
Thanks @harinij
